### PR TITLE
Add datetime for resource attribute type

### PR DESCRIPTION
**Pull request by sileht**
This change adds a new type of attribute for resource: datetime.

### CODE REVIEW

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 5e94a3e00068f20b183b4d2c0b26015627e59904

File: gnocchi/indexer/sqlalchemy_extension.py [hunk 1]
```diff
@@ -54,3 +54,17 @@ class NumberSchema(resource_type.NumberSchema, SchemaMixin):
 
 class BoolSchema(resource_type.BoolSchema, SchemaMixin):
     satype = sqlalchemy.Boolean
+
+
+class DatetimeSchema(resource_type.DatetimeSchema, SchemaMixin):
+    @property
+    def satype(self):
+        # FIXME(sileht): I don't get how we have a circular dependencies here
```
Comment (chungg) [inline gnocchi/indexer/sqlalchemy_extension.py]: i'll be honest, this seems concerning.

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 531efa459e576ea2eae8d49f31e1b4c6bb3a6b56

File: gnocchi/indexer/sqlalchemy_extension.py (hunk 1 above)
Comment (sileht) [reply gnocchi/indexer/sqlalchemy_extension.py]: I fixed it.

**Comment by sileht** [comment]
Pull-request updated, HEAD is now b3d02e2e3fe4f165caf1d86847e1972cea4f2be9

**Comment by sileht** [comment]
Pull-request updated, HEAD is now cd46e342460931cd90b5167b530d08334863f52a

**Pull request merged**